.scrub.sh

```diff
@@ -1,16 +1,10 @@
 #!/bin/bash
-# scrub.sh uses the autopep8 tool to clean up whitespace and other small bits
+# automatic code formatting
 
-# E261 = double space before inline comment
-# E501 = don't squeeze lines to fix max length
-# E302 = don't go crazy with the double whitespace between funcs
-# E401 = don't put imports on separate lines
-# E305 = don't put two blank lines after the last function
-# E309 = don't put a blank line after class declaration
-# E731 = don't assign a lambda expression check.
-
-autopep8 \
-    --in-place --recursive --aggressive \
-    --ignore E501,E302,E261,E401,E305,E309,E731 \
-    --exclude *.html \
-    ubr/
+if [ -e venv/bin/python3.6 ]; then
+    pip install "black~=19.3b0"
+    black ubr/ --target-version py34
+else
+    # seeing this on new 16.04 lax instances
+    echo "black requires Python 3.6+"
```

Comment:
this script should never be used on `lax` instances but only during development right? Add `exit 1`?

Reply:
copy+paste error. I'll fix it up.